src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -270,60 +270,20 @@ public function getName()
      *
      * @return string One of: unknown, dev, stable, eom, eol
      */
-    private function requestSymfonyState()
+    private function calculateSymfonyState()
```

Comment:
Agree. "calculate" reminds me of operations with some input. "Determine" is more suitable for time dependent operation.